.github/workflows/build-wheels.yml

```diff
@@ -142,5 +144,6 @@ jobs:
           repository-url: https://test.pypi.org/legacy/
 
       - name: Upload to pypi
+        # If we tagged a version release it on PyPI
```

Comment:
Which one takes precedence here?

Reply:
On release tags upload to both test and pypi. Perhaps I should change it to only upload to PyPI on release. What do you think?

Reply:
Oh, sorry - I mean if we set `SETUPTOOLS_SCM_PRETEND_VERSION_FOR_TILEDB` then we should probably skip the upload by default to be safe?

Reply:
Yes, I can definitely do that. What do you think about also adding another input to `workflow_dispatch` called `upload_to_test_pypi` that would be a boolean value and if we set it to true it would upload otherwise not. This would be the only way to upload to test pypi. Then release to normal pypi would only happen on tag and only if pretend version is not set.

Reply:
Sounds good!

Reply:
Then the last commit implemented it that way